gcloud/bigtable/happybase/table.py

```diff
@@ -26,10 +26,18 @@
 from gcloud.bigtable.column_family import GCRuleIntersection
 from gcloud.bigtable.column_family import MaxAgeGCRule
 from gcloud.bigtable.column_family import MaxVersionsGCRule
+from gcloud.bigtable.happybase.batch import _get_column_pairs
```

Comment:
You're advocating for reinstatement of `bigtable._helpers` for any helper that gets used in more than one place?

The other one was / is egregious (using a shared sentinel), though this one is justified.